apps/api/src/app/controllers/__tests__/createTestModule.ts

```diff
@@ -245,6 +229,9 @@ export default async (
         ...userOptions,
     };
 
+    // shouldn't we just call this on the data finder service when we initialize the test module?
+    bootstrapDynamicTypes(dataClassProviders);
```

Comment:
Let's try this and see if we can't remove the extra calls to `DynamicDataFinderService.bootstrapDynamicTypes`.

Reply:
Done. This was only possible where `setupIntegrationTest` is used.